src/client/voice/dispatcher/StreamDispatcher.js

```diff
@@ -73,10 +73,10 @@ class StreamDispatcher extends Writable {
       this._setSpeaking(0);
     });
 
-    if (typeof volume !== 'undefined') this.setVolume(volume);
+    this.setVolume(volume);
```

Comment:
Are you sure this change will not cause issues? This would probably even emit the volume change event with undefined

Reply:
Volume change is defined as 1 above in the code when it's extracted, so no matter what it can't be undefined.